### PR TITLE
Add benchmark script

### PR DESCRIPTION
**Pull request by rluvaton**
Moved from https://github.com/nodejs/benchmarking/blob/59861388c1c66a3b91b2696a9c1709f4767be672/experimental/benchmarks/community-benchmark/run.sh

Ref: https://github.com/nodejs/build/issues/3657#issuecomment-2041981609

Next pr will add the CPU pinning

### CODE REVIEW

File: benchmarks/community-benchmark/run.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	FILTER="--filter ${FILTER}"
+fi
+if [ -n "$RUNS" ]; then
+	RUNS="--runs ${RUNS}"
+fi
+# run benchmark
+fileName=output`date +%d%m%y-%H%M%S`.csv
+echo "Output will be saved to $fileName"
+pwd
+./node-master benchmark/compare.js --old ./node-master --new ./node-pr $FILTER $RUNS -- $CATEGORY | tee $fileName
```
Comment (mcollina) [inline benchmarks/community-benchmark/run.sh:104]: Please add the CPUSET instruction.

**Review by mcollina — CHANGES_REQUESTED** [review]
The git history is quite odd, could you fix it?

File: benchmarks/community-benchmark/run.sh (hunk 1 above)
Comment (rluvaton) [reply benchmarks/community-benchmark/run.sh:104]: I wanted to add in different pr, just creating this with the history of the original file

Comment (rluvaton) [reply benchmarks/community-benchmark/run.sh:104]: fixed git history

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**